docs/html/development/release-process.rst

```diff
@@ -65,6 +65,15 @@ their merits.
   ``pip._internal.utils.deprecation.deprecated``. The function is not a part of
   pip's public API.
 
+.. _`Supported Versions`:
```

Comment:
Let's not add this intersphinx reference -- nothing in our docs use this.

Reply:
done!